src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -144,7 +144,7 @@ export const Sidebar = () => {
 						</Button>
 
 						<Button className="hover:bg-green-200" href="/intro" title={t("intro")}>
-							<IconSvg className="size-7 transition" src={HelpIcon} />
+							<IconSvg className="size-5.5 ml-0.5 transition" src={CircleQuestionIcon} />
```

Comment:
You don't have 5.5 size in `tailwind.config.cjs`

Reply:
added